packages/helper-grammar-regex-collection/index.js

```diff
@@ -159,7 +159,29 @@ export const JAVA_IMPORT = regex`
   import
   \s
   (?<$1>
-    javax? # For now, we support java core packages only
+    (
+      javax?
+      |
+      org\.springframework
+      |
+      io\.spring
+      |
+      sparklr\.common
+      |
+      org\.hamcrest
+      |
+      com\.fasterxml
+      |
+      org\.junit
+      |
+      org\.mockito
+      |
+      org\.apache\.log4j
+      |
+      org\.slf4j
+      |
+      org\.apache\.commons\.logging
+    )
```

Comment:
Since we only support a really small portion of java imports, I went with whitelisting the supported packages. Everything else would raise wrong expectations when every import is clickable.